spec/book_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../book'
+
+describe Book do
+  describe '#initialize' do
+    it 'creates a new instance of Book with the given title and author' do
+      title = 'Title'
+      author = 'Author'
+
+      book = described_class.new(title, author)
+
+      expect(book.title).to eq(title)
+      expect(book.author).to eq(author)
+      expect(book.rentals).to be_empty
+    end
+  end
```

Comment:
- [ ] As required for the project to have unit tests for all the methods in all the classes, I think it would be better if you add tests to the `rent_book` method in your `Book` class. And for best practices, I suggest you put each specific test in an `it` block like you did in other files.
```ruby
it "Should test the rent_book method" do
# Your code here
end
```